server/src/main/java/org/elasticsearch/index/mapper/DynamicKeyFieldMapper.java

```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-package org.elasticsearch.index.mapper;
-
-import org.elasticsearch.index.analysis.NamedAnalyzer;
-
-/**
- * A field mapper that supports lookup of dynamic sub-keys. If the field mapper is named 'my_field',
```

Comment:
It looks like we dropped this javadoc, it'd be nice to move it to `DynamicFieldType` instead.